.gitignore

```diff
@@ -1 +1,4 @@
 .DS_store
+students/mattcasari/template.py
+students/mattcasari/lesson02/.vscode/settings.json
+students/mattcasari/lesson03/.vscode/settings.json
```

Comment:
please remove these changes - please do not make changes outside of your student directory. 

you can add .gitignore in your own directory.